### ISSUE TITLE
Credentials?

### ISSUE THREAD

**Issue by zag2me** [metabrainz/musicbrainz-vm#2]
Just trying to give this a test, have the credentials changed at all?

Tried vm/musicbrainz with no luck...

![image1](https://cloud.githubusercontent.com/assets/1050512/17807051/34e75ea0-6600-11e6-8503-76d0c5a8d6cd.jpg)

**Comment by mayhem** [comment]
Try vagrant/vagrant

**Comment by zag2me** [comment]
thx works, might be worth updating the readme

**Comment by zuperjotmeil** [comment]
I have managed to login but when I try to initiate replication fails with:
vagrant@vagrant-ubuntu-trusty-64:~$ bin/replicate now
Error response from daemon: Container d01c6166ee03344d2f0fff5a5435b634c486cf252f55836824085e0129261f3e is not running

I have manually edited the DBDefs.pm file with the token .

Any help? How could start the docker? I have tryed with:
vagrant@vagrant-ubuntu-trusty-64:~$ docker-exec
docker-exec: command not found

Do I need to install vagrant in the VM?

Thanks in advance for your help.
